doc/conf.py

```diff
@@ -55,6 +55,13 @@
 
 primary_domain = 'hs'
 
+# SPHINX_HADDOCK_DIR comes from `combined-plutus-apps-haddock.nix` derivation, which
```

Comment:
@michaelpj  please review this file and tell me if it makes sense.

> However `combine-haddock.nix` only generates objects.inv files for the project's packages.
See https://github.com/input-output-hk/plutus-apps/blob/0a3db995df1d733ba8af8223bc8d3725218be66e/__std__/cells/plutus-apps/library/combine-haddock.nix#L80

Reply:
I guess. TBH I don't really remember. It might be possible to just get `combine-haddock` to include more stuff?